### PR TITLE
feat: Gemini Supervised Tuning for Q&A

### PR DESCRIPTION
**Pull request by erwinh85**
# Description

Thank you for opening a Pull Request!
Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

- [X] Follow the [`CONTRIBUTING` Guide](https://github.com/GoogleCloudPlatform/generative-ai/blob/main/CONTRIBUTING.md).
- [X] You are listed as the author in your notebook or README file.
  - [X] Your account is listed in [`CODEOWNERS`](https://github.com/GoogleCloudPlatform/generative-ai/blob/main/.github/CODEOWNERS) for the file(s).
- [X] Make your Pull Request title in the <https://www.conventionalcommits.org/> specification.
- [X] Ensure the tests and linter pass (Run `nox -s format` from the repository root to format).
- [X] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: gemini/tuning/Gemini_Supervised_Tuning_for_Q&A.ipynb [hunk 1]
```diff
@@ -0,0 +1,923 @@
+{
```
Comment (holtskinner) [inline gemini/tuning/Gemini_Supervised_Tuning_for_Q&A.ipynb:1]: <p>Remove extra <code>\</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/606/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (holtskinner) [inline gemini/tuning/Gemini_Supervised_Tuning_for_Q&A.ipynb:1]: <p>Fix formatting</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/606/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Comment (holtskinner) [inline gemini/tuning/Gemini_Supervised_Tuning_for_Q&A.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #2.    from vertexai import generative_models
</pre><p>This extra import shouldn't be needed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/606/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Comment (holtskinner) [inline gemini/tuning/Gemini_Supervised_Tuning_for_Q&A.ipynb:1]: <p>Can you explain the random_state and why it's 42? (If relevant)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/606/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Comment (holtskinner) [inline gemini/tuning/Gemini_Supervised_Tuning_for_Q&A.ipynb:1]: <p>For the bullet points, I'd recommend code font</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/606/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Comment (holtskinner) [inline gemini/tuning/Gemini_Supervised_Tuning_for_Q&A.ipynb:1]: <p>You might be able to write the JSONL file directly to GCS in Pandas if you pass the <code>gs://</code> path to <code>to_json()</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/606/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Comment (holtskinner) [inline gemini/tuning/Gemini_Supervised_Tuning_for_Q&A.ipynb:1]: <p>Could you add another example to show the difference in output for the foundation model and the tuned model?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/606/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='66'/>

**Review by holtskinner — APPROVED** [review]
Minor Comments, LGTM

File: gemini/tuning/Gemini_Supervised_Tuning_for_Q&A.ipynb (hunk 1 above)
Comment (erwinh85) [reply gemini/tuning/Gemini_Supervised_Tuning_for_Q&A.ipynb:1]: <p>We're working on adding Eval. I will leave this open and add eval later. Thanks. </p>

**Pull request closed without merging**